setup.py

```diff
@@ -42,6 +42,9 @@
             'start-artman-conductor = artman.cli.conductor:start',
         ],
     },
+    scripts=[
+        'test/smoketest_artman.py',
```

Comment:
Just to make sure I follow the release process. This script will be executed every time a new artman pypi package (plus artman docker image) gets released?